### PR TITLE
Protractor e2e testing with hidden elements

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @schmierkov @mauro-oto,

This PR explains how to interact with hidden elements in Protractor. 

Please check it out and let me know your thoughts. 

Thanks!

### CODE REVIEW

File: _posts/2015-11-23-how-to-hide-elements-for-protractor-e2e-tests.markdown [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  display: none;
+}
+
+{% endhighlight %}
+
+Which caused this problem:
+
+    Failed: Wait timed out after 100015ms
+
+Workarounds include _displaying it_, _interacting with it_, _hidding it_ again, which I didn't like.
```
Comment (mauro-oto) [inline _posts/2015-11-23-how-to-hide-elements-for-protractor-e2e-tests.markdown]: There's a typo in `hidding`, should be `hiding`.

File: _posts/2015-11-23-how-to-hide-elements-for-protractor-e2e-tests.markdown [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  <input class="btn btn-default" style="opacity:0;height:0px;" accept=".csv"  id="geofence_file_input" type="file">
+  <a class="btn btn-default" id="textbox-for-geofencefile" ng-click="selectFile(this)">Select File</a>
+  <span ng-if="LineItemForm.augmentations.geofence.file">{{selectedFilename()}}</span>
+</div>
+
+{% endhighlight %}
+
+Or with CSS:
+
+{% highlight css %}
```
Comment (mauro-oto) [inline _posts/2015-11-23-how-to-hide-elements-for-protractor-e2e-tests.markdown]: I like this syntax, I used something like (not sure if Github will escape this): 

```css
#geofence_file_input { }
```

In a past post, so it's three backticks followed by the language to highlight. Not sure which is best or even if there's any difference, but just thought I'd mention it as an alternative.

**Comment by mauro-oto** [comment]
@etagwerker great! :+1: Left a couple of comments to review.

File: _posts/2015-11-23-how-to-hide-elements-for-protractor-e2e-tests.markdown (hunk 2 above)
Comment (etagwerker) [inline _posts/2015-11-23-how-to-hide-elements-for-protractor-e2e-tests.markdown]: @mauro-oto Good to know! Both work perfectly.

**Comment by etagwerker** [comment]
@mauro-oto Done!

**Comment by mauro-oto** [comment]
@etagwerker Looks good! :+1:

**Pull request merged**